scripts/update_localization.rb

```diff
@@ -73,6 +76,15 @@
                       "messages#{@options[:pot_suffix]}"
     end
   end
+
+  # Remove trailing space
+  OS.windows?
+    info "Windows detected. Removing trailing space..."
+
+    LOCALES.each do |locale|
+      target = locale + @options[:po_suffix]
+      File.write(target, File.read(target).gsub(/# \n/,"#\n"))
```

Comment:
I think you should switch the code to use a line by line approach to find the trailing whitespace, and *only* write the new contents to disk if there is a change.

Reply:
As above.

Reply:
Other people won't be happy if their disk is uselessly worn down by rewriting 30 files that don't change.